feature_engine/discretisation/arbitrary.py

```diff
@@ -7,9 +7,10 @@
 
 from feature_engine.base_transformers import BaseNumericalTransformer
 from feature_engine.validation import _return_tags
+from feature_engine.discretisation import BaseDiscretiser
```

Comment:
to import it when it is not in the init, you need to do:

`from feature_engine.discretisation.base_discretiser import BaseDiscretiser`